src/include/cluster/helps.rb

```diff
@@ -50,7 +50,13 @@ def initialize_cluster_helps(include_target)
             "<p><b><big>Port</big></b><br>Specifies TCP port of qnetd server. Default is 5403.</p>\n" +
             "<p><b><big>TLS</big></b><br>Can be one of 'on', 'off' or 'required' and specifies if tls should be used. 'on' means a connection with TLS is attempted first, but if the server doesn't advertise TLS support then non-TLS will be used. 'off' is used then TLS is not required and it's then not even tried. This mode is the only one which doesn't need a properly initialized NSS database. 'required' means TLS is required and if the server doesn't support TLS, qdevice will exit with error message. 'on' need manually change, refer to corosync-qdevice's man page for more details. Default is 'off' in yast.</p>\n" +
             "<p><b><big>Algorithm</big></b><br>Decision algorithm. Can be one of the 'ffsplit' or 'lms'.  (Actually there are also 'test' and '2nodelms', both of which are mainly for developers and shouldn't be used for production clusters, so yast will convert to 'ffsplit' automatically). For a description of what each algorithm means and how the algorithms differ see their individual sections.  Default value is ffsplit.</p>\n" +
-            "<p><b><big>Tie breaker</big></b><br>Can be one of 'lowest', 'highest' or 'valid_node_id' (number) values. It's used as a fallback if qdevice has to decide between two or more equal partitions. 'lowest' means the partition with the lowest node id is chosen. 'highest' means the partition with highest node id is chosen. And 'valid_node_id' means that the partition containing the node with the given node id is chosen. Default is 'lowest'.</p>\n"
+            "<p><b><big>Tie breaker</big></b><br>Can be one of 'lowest', 'highest' or 'valid_node_id' (number) values. It's used as a fallback if qdevice has to decide between two or more equal partitions. 'lowest' means the partition with the lowest node id is chosen. 'highest' means the partition with highest node id is chosen. And 'valid_node_id' means that the partition containing the node with the given node id is chosen. Default is 'lowest'.</p>\n" +
+            "<p><b><big>Qdevice Heuristics</big></b><br>Subsection of Qdevice. Heuristics are set of commands executed locally on startup, cluster membership change, successful connect to corosync-qnetd and optionally also at regular times. Commands are executed in parallel. When *all* commands finish successfully (their return error code is zero) on time, heuristics have passed, otherwise they have failed. The heuristics result is sent to corosync-qnetd and there it's used in calculations to determine which partition should be quorate.</p>\n" +
```

Comment:
Quite long lines :)

It has been a bit difficult to check them.

We usually _split_ those long help lines text. For example


```ruby
    
"<p><b><big>Qdevice Heuristics</big></b><br>Subsection of Qdevice. " \
"Heuristics are set of commands executed locally on startup, cluster membership change, " \
"successful connect to corosync-qnetd and optionally also at regular times. Commands are " \
"executed in parallel. When *all* commands finish successfully (their return error code is zero) " \
"on time, heuristics have passed, otherwise they have failed. The heuristics result is sent to " \
"corosync-qnetd and there it's used in calculations to determine which partition should be quorate.</p>"
```

which helps a lot of not only during review, also when reading/changing them

Reply:
Fixed in new commit.